my-contacts/src/Form.js

```diff
@@ -0,0 +1,64 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {useStyles} from "react-treat";
+import * as styleRefs from "./Form.treat";
+
+const Form = (props) =>
+ {
+     const styles = useStyles(styleRefs);
+
+     const [image, setImage] = useState("");
+     const [firstName, setFirstName] = useState("");
+     const [lastName, setLastName] = useState("");
+     const [companyName, setCompanyName] = useState("");
+     const [phoneNumber, setPhoneNumber] = useState("");
+     const [email, setEmail] = useState("");
+
+     const submit = () => {
+         let newContact = {
+             image: image,
+             firstName: firstName,
+             lastName: lastName,
+             companyName: companyName,
+             phone: phoneNumber,
+             email: email
+         };
+
+         props.addContact(newContact);
+     };
+
+    return (
+        <div className={styles.container}>
+            <label className={styles.label}>Image
+                <input className={styles.input} value={image} onChange={e => setImage(e.target.value)}/>
```

Comment:
Does this work? I assumed you would need to set `type='image'` on the input

Reply:
This input just takes a url for the image so I don't think it needs the `type='image'`?

Reply:
OH I see, We can chat tomoz abt the type input, but basically lets you chose an image from the file system.